src/ExpressionEvaluator/VisualBasic/Source/ExpressionCompiler/SymUnmanagedReaderExtensions.vb

```diff
@@ -0,0 +1,79 @@
+﻿Imports System.Collections.Immutable
```

Comment:
Fixed, but I couldn't find the file I copied it from (which, presumably, also lacks a copyright header).